lua/wire/client/hlzasm/hc_output.lua

```diff
@@ -143,7 +143,6 @@ function HCOMP:OutputLibrary(block)
     for index,value in ipairs(block.Data) do
       if istable(value)then -- Data is a constant expression
         block.Data[index] = self:PrintTokens(value)
-        block.Data[index] = v or self.Settings.MagicValue
```

Comment:
(For anybody else reading this code - at this point `v` is undefined and `self.Settings.MagicValue` is [defined](https://github.com/wiremod/wire/blob/09e8f9573b4411ced0f188cdd314b5ed387d4074/lua/wire/client/hlzasm/hc_compiler.lua#L193) to be a constant that signals invalid bytecode.